.travis.yml

```diff
@@ -1,14 +1,22 @@
 language: python
 python:
   - "2.7"
-branches:
-  only:
-  - /.*/
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+    - os: osx
+      language: generic
 env:
   - SYSL_PLANTUML=http://www.plantuml.com/plantuml
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
```

Comment:
[[ ... ]] is a bashism. Without anything specifying bash, you should stick to posix shell constructs:

if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew update; fi

Note the single equals sign instead of double-equals (which is also a bash-ism)

Reply:
Done

Reply:
You missed the last line of my comment, re: = vs ==. Posix does not support ==, only =

Reply:
Apologies! I've got great intentions to read reviews more carefully from here onwards.
Done.